### PR TITLE
Twine Importer UI: JSON Compatibility

### PR DESCRIPTION
**Pull request by cpimhoff**
This update changes the GUI to accept JSON files instead of HTML, and fixes a few bugs.

This PR also comes bundles with a simple test JSON file (which we can remove later).
### Changes
- Twine Importer GUI (and native file picker) now only accepts files with _.json_ extensions.
- `TwineImporterUI.SendToImporter(file)` now bridges to the `TwineJsonParser.ImportFile(file)`
- Right clicking a json asset in the asset browser and clicking "Import Twine Data..." now correctly auto populates the file field.
- Fixed issue where paths from a PC directory style were not validated correctly.

### CODE REVIEW

**Comment by cpimhoff** [comment]
PR #17, has been closed in favor of this. I now realize there was a better way to _replace_ a PR, but alas, hindsight is 20/20.

File: Project/Assets/Editor/Twine Importer/TwineImporterUI.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ public class TwineImporterUI : EditorWindow {
 		TextAsset selectedFile = null;
 		if (Selection.activeObject != null) {
 			var filePath = AssetDatabase.GetAssetPath (Selection.activeObject);
-			if (Path.GetExtension (filePath) == "html") {
+			if (Path.GetExtension (filePath).Contains ("json")) {
```
Comment (grahamearley) [inline Project/Assets/Editor/Twine Importer/TwineImporterUI.cs:25]: Why the switch from `==` to `Contains`?

**Review by grahamearley — APPROVED** [review]
Looks nice.
@brennan724 Take note of these changes -- they might affect what you were working on because they make the `TwineJsonParser` script use different methods

**Comment by grahamearley** [comment]
but (re: my previous comment to @brennan724)  this is definitely a step forward and we should go off this one rather than my dummy code.

**Review by brennan724 — APPROVED** [review]
Yay!

**Pull request merged**